README.md

```diff
@@ -51,63 +59,62 @@ To run Zilliqa, we recommend the following minimum system requirements:
-* `build-essential`
-* `Boost` 
-* `CMake`
-* `clang-format`
-* `git`
-* `JsonCpp`
-* `json-rpc-cpp`
-* `LevelDB`
-* `OpenSSL`
-* `pkg-config`
-* `libevent`
-* `miniupnpc`
-* `opencl`
-* `Protobuf`
-* Dependencies only required for builds with code coverage enabled
-    * `gcov`
-    * `lcov`
-* Dependency only required for builds on Travis-CI
-    * `curl`
-
-For Ubuntu 16.04, you can use the following commands (or refer to `./scripts/ci_install_deps.sh`) to install the dependencies:  
-
-```bash
-sudo apt-get update
-```
-
-```bash
-sudo apt-get install git clang-format-5.0 clang-tidy-5.0 clang-5.0 libboost-system-dev libboost-filesystem-dev libboost-test-dev libssl-dev libleveldb-dev libjsoncpp-dev libsnappy-dev cmake libmicrohttpd-dev libjsonrpccpp-dev build-essential pkg-config libevent-dev libminiupnpc-dev ocl-icd-opencl-dev libprotobuf-dev protobuf-compiler
-```
-
-For Mac OS X (experimental), you can use the following command to install the dependencies:  
-```bash
-brew install pkg-config jsoncpp leveldb libjson-rpc-cpp libevent miniupnpc
-```
-
-# Running Zilliqa locally (using 10 shard nodes and DS node locally)  
+
+* Ubuntu 16.04:
+
+    ```bash
+    sudo apt-get update
+    sudo apt-get install git clang-format-5.0 clang-tidy-5.0 clang-5.0 libboost-system-dev \
+        libboost-filesystem-dev libboost-test-dev libssl-dev libleveldb-dev libjsoncpp-dev \
+        libsnappy-dev cmake libmicrohttpd-dev libjsonrpccpp-dev build-essential pkg-config \
+        libevent-dev libminiupnpc-dev ocl-icd-opencl-dev libprotobuf-dev protobuf-compiler
+    ```
+
+* macOS (experimental):
+
+    ```bash
+    brew install pkg-config jsoncpp leveldb libjson-rpc-cpp libevent miniupnpc
+    ```
+
+## Running Zilliqa locally
+
 1. Build Zilliqa from the source.  
-` ./build.sh`
+
+    ```
+    ./build.sh
+    ```
 
 2. Run the local testnet script in `build` directory
-`cd build && ./tests/Node/test_node_simple.sh`  
+
+    ```
+    cd build && ./tests/Node/test_node_simple.sh
+    ```
 
 3. Logs of each node can be found at `./local_run`
 
 4. To terminate Zilliqa,   
-`pkill zilliqa` 
+    ```
+    pkill zilliqa
+    ``` 
 
-If you wish to use CUDA supported GPU for PoW, please download and install CUDA package from [NVIDIA offical webpage](https://developer.nvidia.com/cuda-downloads). You may need to reboot your PC for the installation take effect. After which, please use the following build option to build Zilliqa with CUDA support.  
+## GPU mining
+The OpenCL support is build into the Zilliqa by default. If you have a graphics card with AMD GPU, you can use it for PoW already.
+
+If you wish to use CUDA supported GPU for PoW, please download and install CUDA package from [NVIDIA offical webpage](https://developer.nvidia.com/cuda-downloads). You may need to reboot your PC for the installation take effect. After which, please use the following build option to build Zilliqa with CUDA support.
 `./build.sh cuda`
```

Comment:
@ShengguangXiao can you sqaush your commits into one?

Also, let's put `./build.sh cuda` in a code block like this

```
./build.sh cuda
```